### ISSUE TITLE
Pynest version retrieval not standard

### ISSUE THREAD

**Issue by morales-gregorio** [nest/nest-simulator#1348]
Hello!

I was trying to retrieve the nest version from pynest and noticed that this is done by calling a function `nest.version()`. 

[PEP396 recommends](https://www.python.org/dev/peps/pep-0396/)  adding it as a module [dunder](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names):  `nest.__version__`. Even though PEP396 is deferred (not sure why or by which other PEP entry), to me this is more intuitive and feels quite standard for all other python packages I have used, some examples:
```
>>> numpy.__version__
'1.17.3'
>>> scipy.__version__
'1.3.1'
>>> matplotlib.__version__
'3.1.2'
>>> pandas.__version__
'0.25.3'
>>> seaborn.__version__
'0.9.0'
```

Also the version should probably just be a string of the version number and not contain the package name in it like it is now, e.g. `'Nest 2.18.0'` --> `'2.18.0'`.

I encountered this issue together with issue #1347 and they can probably be solved together.

**Comment by jougs** [comment]
@morales-gregorio: Now that #1347 fixes the actual version returend, could you please file a PR that adds an appropriate `__version__()` function with the `@property` decorator? Thanks!

**Comment by morales-gregorio** [comment]
Alright, I will try to!

**Comment by stinebuu** [comment]
@morales-gregorio Are there any updates on this?

**Comment by morales-gregorio** [comment]
oh yes, I had difficulties building NEST in my local machine (my paths are messed up apparently) and then completely forgot about it. I will come back to this, the actual updates seem pretty straightforward, I will open the PR as soon as possible.

**Comment by Silmathoron** [comment]
@jougs Should we actually keep an equivalent of the previous ``version()`` function that would return kernel information?
Like ``nest.version()`` is a non-standard thing which gives more info and ``nest.__version__`` is the standard python version?

**Comment by jougs** [comment]
I think `version()` can be removed once the standard `__version__` is in place. I don't see any reason for keeping it other than confusing users, which is usually not what we want :wink:
